go.sum

```diff
@@ -320,6 +320,8 @@ github.com/stretchr/testify v1.8.2 h1:+h33VjcLVPDHtOdpUCuF+7gSuG3yGIftsP1YvFihtJ
 github.com/tonglil/buflogr v1.0.1 h1:WXFZLKxLfqcVSmckwiMCF8jJwjIgmStJmg63YKRF1p0=
 github.com/vertica/vcluster v1.1.0 h1:IYtCMKKTQUmLqyPfp5GRYzKiuB3LwY0GKwSjlqint/Y=
 github.com/vertica/vcluster v1.1.0/go.mod h1:mr87BZeipS9HMaL1vKST7lbJ1+SytCbruTF+8tsdK78=
+github.com/vertica/vcluster v1.1.1-0.20240125183229-a7b16dc76c02 h1:a6NE/ux7oIt3aXKSl4exc448E7/Exh5LEpTDJbB7fsY=
```

Comment:
Make sure you do a go mod tidy. This will clean out the old vcluster package.

Reply:
done.